### PR TITLE
Upgrading to 6.2.1

### PR DESCRIPTION
**Pull request by biorus**
- Changed Liferay version
- Changed the category and the weight for control-panel-entry
- Added TLD dependencies in liferay-plugin-package.properties
- Removed the parameters yearRangeEnd and yearRangeStart from
  liferay-ui:input-date according to the new TLD
- Raised minute interval to 30

### CODE REVIEW

File: src/main/webapp/WEB-INF/liferay-plugin-package.properties [hunk 1]
```diff
@@ -8,7 +8,14 @@ page-url=http://emdev.ru
 author=EmDev LLC
 portal-dependency-jars=\
     commons-lang.jar
-liferay-versions=6.1.1+,6.1.20+
+liferay-versions=6.2.1+,6.2.10+
```
Comment (akakunin) [inline src/main/webapp/WEB-INF/liferay-plugin-package.properties]: should be 6.2.0+,6.2.10+

File: pom.xml [hunk 2]
```diff
@@ -1,121 +1,121 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>ru.emdev</groupId>
+	<artifactId>extended-auth-portlet</artifactId>
+	<packaging>war</packaging>
+	<name>Extended Authentication Settings hook</name>
+	<version>6.2.1.1</version>
+
+	<properties>
+		<liferay.version>6.2.1</liferay.version>
```
Comment (akakunin) [inline pom.xml]: I think better to use latest 6.2.3 here

**Comment by akakunin** [comment]
Hi! Thank you for pull request. There are only two small comments to fix.

**Comment by biorus** [comment]
Hi,

I'll change those but it'll take me a bit of time. I chose these version
because I'm mainly working with EE.

Regards,
Jose

2015-09-10 15:23 GMT+01:00 Alexey Kakunin notifications@github.com:

**Comment by akakunin** [comment]
you do not need to switch to CE - just change numbers in code - it should be ok - I will test on CE from my side

**Pull request merged**